### DISCUSSION TITLE
Absolute positioning for block elements

### DISCUSSION THREAD

**Discussion by sohamb91** [felixmariotto/three-mesh-ui#272] [Q&A] [answered]
I am trying to recreate a 2D panel from component co-ordinates captured from unity. I have applied all unity -> threejs transformations. However due to the flex nature, the world space coordinates captured from unity are not being applied to the block elements. Any better way to approach this problem? Is there any way the block elements an accept "absolute" positioning?

**Comment by swingingtom** [comment, marked as answer]
Hello @sohamb91,
Have you already tried this https://felixmariotto.github.io/three-mesh-ui/#manual_positioning ?

**Reply by sohamb91** [reply]
I had not tried this. thank you so much!
